source/backups.rst

```diff
@@ -172,54 +180,100 @@ message in the operator logs.
+   More details about secrets can be found in :ref:`users.system-users`.
 
-Restoring without point-in-time recovery
-****************************************
+Following things are needed to restore a previously saved backup:
 
-Following steps are needed to restore a previously saved backup:
+* Make sure that the cluster is running.
 
-1. First of all make sure that the cluster is running.
+* Find out correct names for the **backup** and the **cluster**. Available
+  backups can be listed with the following command:
 
-2. Now find out correct names for the **backup** and the **cluster**. Available
-   backups can be listed with the following command:
+  .. code:: bash
 
-   .. code:: bash
+     kubectl get psmdb-backup
 
-      kubectl get psmdb-backup
+  .. note:: Obviously, you can make this check only on the same cluster on
+     which you have previously made the backup.
 
-   And the following command will list available clusters:
+  And the following command will list available clusters:
 
-   .. code:: bash
+  .. code:: bash
 
-      kubectl get psmdb
+     kubectl get psmdb
+
+.. _backups-no-pitr-restore:
+
+Restoring without point-in-time recovery
+****************************************
 
-3. When both correct names are known, run the actual restoration process:
+When the correct names for the backup and the cluster are known, backup
+restoration can be done in the following way.
+
+1. Set appropriate keys in the `deploy/backup/restore.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/backup/restore.yaml>`_ file.
+
+   * set ``spec.clusterName`` key to the name of the target cluster to restore
+     the backup on,
+   * if you are restoring backup on the *same* Kubernetes-based cluster you have
+      used to save this backup, set ``spec.backupName`` key to the name of your
+      backup,
+   * if you are restoring backup on the Kubernetes-based cluster *different*
+     from one you have used to save this backup, set ``spec.backupSource``
+     subsection instead of ``spec.backupName`` field to point on the appropriate
+     S3-compatible storage. This ``backupSource`` subsection should contain
+     a ``destination`` key equal to the s3 bucket with a special ``s3://``
+     prefix, followed by necessary S3 configuration keys, same as in
+     ``deploy/cr.yaml`` file:
+
+     .. code-block:: yaml
+
+        ...
+        backupSource:
+          destination: s3://S3-BUCKET-NAME/BACKUP-NAME
+          s3:
+            credentialsSecret: my-cluster-name-backup-s3
+            region: us-west-2
+            endpointUrl: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
+   * you can also use a ``storageName`` key to specify the exact name of the
+     storage (the actual storage should be already defined in the
+     ``backup.storages`` subsection of the ``deploy/cr.yaml`` file):
+
+     .. code-block:: yaml
+
+        ...
+        backupSource:
+          storageName: s3-us-west
```

Comment:
Where do I specify the backup name in this case?

Reply:
@fiowro 

```yaml
  ...
  backupSource:
    destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```